docs/import-cli.rst

```diff
@@ -260,6 +264,59 @@ Note: Connecting over SSH is necessary only if you intend to import in-place. If
 
 -  Preparation of the .csv or .tsv file.
 
+Concatenate the CLI imports with post-import steps
+==================================================
+
+The following example shows how to do the import on CLI and follow-up operations like rendering and metadata import in one step.
+
+See (reference) for rendering and (reference) for metadata.
+
+In many cases, the rendering and metadata import is preferable to be done separately, as the visual checking of the imported images might be crucial for further rendering and metadata import, see :doc:`render` and :doc:`metadata` for details on this. 
+
+Further, the images you are importing might need a range or different rendeing settings, not just one settings for all of them. Also for this case, the step-by-step approach, first importing the images, then only deciding on the rendering strategy and preparing the `renderingdef.yml` files, is preferable.
+
+Nevertheless, there are cases which do not need visual checks and use a single rendering for all images, for which a streamlined sequence of commands is offered below which will perform all three steps (import, rendering and metadata import) in one single session on the CLI.
+
+Resources
+---------
+
+Additionally to the :ref:`Resources mentioned in the import-cli section<Resources>` and in the :ref:`Setup<Setup>` you will also need the rendering and metadata plugins as mentioned in :doc:`render` and :doc:`metadata`, and possibly the following files:
+
+   - https://github.com/ome/training-scripts/blob/master/maintenance/preparation/renderingdef.yml
```

Comment:
Not specific to this PR but I have been wondering about how the choice is made between references to files living in external repositories vs files within this guide? As a side note, we should probably start using tags than `master`.

Reply:
The choice is dictated by the existence or absence of the particular file in some other resource. The `training-scripts` repo pre-dates the guides, and thus this (present) case is one of re-using of existing content.

I will have a look at the formatting of the reference.

Regarding the `master` vs `tag`, this file will change soon, see header of this PR (ToDo) so I will fix the tag as a follow-up of that ToDo here ?
I think @jburel prefers the citation of the necessary version in the Resources, see QuPath guide.